openvpn-install.sh

```diff
@@ -836,8 +844,13 @@ ifconfig-pool-persist ipp.txt" >>/etc/openvpn/server.conf
 			echo "push \"dhcp-option DNS $DNS2\"" >>/etc/openvpn/server.conf
 		fi
 		;;
+	14) # No DNS
+		echo "No DNS push config"
+		;;
 	esac
-	echo 'push "redirect-gateway def1 bypass-dhcp"' >>/etc/openvpn/server.conf
+	if [[ $DNS != 14 ]]; then
+		echo 'push "redirect-gateway def1 bypass-dhcp"' >>/etc/openvpn/server.conf
+	fi
```

Comment:
You are using the `None` option to connect to your devices without redirecting all the traffic. It's not really the script's goal, so maybe a FAQ would be better? @angristan

Reply:
In my case I want a secure way to access to my servers, it's not my idea to redirect all traffic.
So I assume not wanting a DNS means also not wanting to redirect all traffic. (I believe users can avoid redirecting traffic anyway, but sounds natural to me to also enforce it at server level)
I use `$CLIENT_TEMPLATE_APPEND` option to add specific routes. eg:
```
CLIENT_TEMPLATE_APPEND="
route 172.34.0.0 255.255.0.0
"
```
I will clarify the option. (or perhaps i can split in two options (redirecting and DNS))

Reply:
> In my case I want a secure way to access to my servers, it's not my idea to redirect all traffic.

Again, not really the script's goal but I understand that need, I used to do the same thing.

> So I assume not wanting a DNS means also not wanting to redirect all traffic.

Yes you assume correctly, otherwise it would be a DNS leak.